prboom2/src/r_data.c

```diff
@@ -625,7 +625,7 @@ int R_SetSpriteByIndex(patchnum_t *patchnum, spritenum_t item)
 
 int R_NumPatchForSpriteIndex(spritenum_t item)
 {
-  if (item < 0 || item >= num_sprites)
```

Comment:
This is a "dead" condition because `spritenum_t` has no negative values, and I didn't see anything that cast negative values to it, but I'll double-check.

Reply:
I'd leave this as is just to be safe. Even if nothing is doing it now, it might at some point 😸